tuna/go_fish.py

```diff
@@ -43,8 +43,13 @@ def parse_args():
                       nargs='?',
                       default=Library.MIOPEN,
                       type=Library,
-                      help="Specify library to run",
+                      help="Specify library to run, defaults to MIOpen",
                       choices=Library)
+  parser.add_argument('--add_tables',
+                      dest='add_tables',
+                      action='store_true',
+                      help='Add library specific tables. If no library specified, '\
+                           'MIOpen is implied')
```

Comment:
Perhaps use `mutually_exclusive_group` from `argparse` to address this.

Reply:
Moved to individual lib